### PR TITLE
add command to copy tags from one project to another

### PR DESCRIPTION
**Pull request by ssadedin**
There might be already a good way to do this but I found I often want to copy the tags from one project to another, so I made a simple SeqR management command for that.

Feel free to close if you already have a way to do this but I figured since it's useful for me I'd offer it for what it is worth.

### CODE REVIEW

**Comment by hanars** [comment]
Since most of the tags at this point are the non-project-specific tags and are added to all projects by default and we are trying to minimize the number of project specific tags we have, we've never needed this functionality. However, theres no reason not to have it!

File: seqr/management/commands/copy_project_tags.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models.query_utils import Q
+
+from seqr.models import Project, VariantTagType
+from seqr.model_utils import create_seqr_model
```
Comment (hanars) [inline seqr/management/commands/copy_project_tags.py:5]: This isn't used

File: seqr/management/commands/copy_project_tags.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand, CommandError
```
Comment (hanars) [inline seqr/management/commands/copy_project_tags.py:1]: CommandError isn;t used

File: seqr/management/commands/copy_project_tags.py (hunk 1 above)
Comment (ssadedin) [reply seqr/management/commands/copy_project_tags.py]: have removed the unused imports in latest commits

File: seqr/management/commands/copy_project_tags.py (hunk 2 above)
Comment (ssadedin) [reply seqr/management/commands/copy_project_tags.py]: have removed the unused imports in latest commits

**Pull request merged**